src/main/java/org/littleshoot/proxy/impl/ReadLoggingHandler.java

```diff
@@ -0,0 +1,40 @@
+package org.littleshoot.proxy.impl;
+
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelPromise;
+import io.netty.handler.logging.LogLevel;
+import io.netty.handler.logging.LoggingHandler;
+import io.netty.util.internal.logging.InternalLogLevel;
+import java.net.SocketAddress;
+import org.littleshoot.proxy.RequestTracer;
+
+public class ReadLoggingHandler extends LoggingHandler {
+
+  private RequestTracer requestTracer;
+
+  public ReadLoggingHandler(RequestTracer requestTracer) {
+    super(LogLevel.DEBUG);
+    this.requestTracer = requestTracer;
+  }
+
+  @Override
+  public void connect(ChannelHandlerContext ctx, SocketAddress remoteAddress, SocketAddress localAddress,
+      ChannelPromise promise) throws Exception {
+    this.logger.log(InternalLogLevel.INFO, this.format(ctx, "Connected"));
+    ctx.connect(remoteAddress, localAddress, promise);
+  }
```

Comment:
Thought: Really wish we could integrate vgs-logging here so it dumps the standard fields etc